cli/deployment/utils_test.go

```diff
@@ -42,9 +43,64 @@ func TestReparentPath(t *testing.T) {
 	}
 }
 
+var fileNameTests = []struct {
+	file string
+	name string
+}{
+	{"name", "name"},
```

Comment:
Consider also testing the empty string, spaces, single and double quotes, and other odd characters that sometimes appear in filenames.

Reply:
done